<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableTools</class>
 <widget class="QWidget" name="TableTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>474</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="rowsHeader">
     <property name="text">
      <string>Row</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rowBtn1">
     <property name="text">
      <string>Remove Selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rowBtn2">
     <property name="text">
      <string>Add Here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rowBtn3">
     <property name="text">
      <string>Add on Top</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rowBtn4">
     <property name="text">
      <string>Add Bottom</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="columnsHeader">
     <property name="text">
      <string>Column</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="colBtn1">
     <property name="text">
      <string>Remove Selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="colBtn2">
     <property name="text">
      <string>Add Here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="colBtn3">
     <property name="text">
      <string>Add Left</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="colBtn4">
     <property name="text">
      <string>Add Right</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
